app/src/main/java/de/westnordost/streetcomplete/quests/parcel_locker_brand/AddParcelLockerBrand.kt

```diff
@@ -0,0 +1,31 @@
+package de.westnordost.streetcomplete.quests.parcel_locker_brand
+
+import de.westnordost.streetcomplete.R
+import de.westnordost.streetcomplete.data.osm.geometry.ElementGeometry
+import de.westnordost.streetcomplete.data.osm.mapdata.Element
+import de.westnordost.streetcomplete.data.osm.mapdata.MapDataWithGeometry
+import de.westnordost.streetcomplete.data.osm.mapdata.filter
+import de.westnordost.streetcomplete.data.osm.osmquests.OsmFilterQuestType
+import de.westnordost.streetcomplete.data.user.achievements.EditTypeAchievement.CITIZEN
+import de.westnordost.streetcomplete.osm.Tags
+
+class AddParcelLockerBrand : OsmFilterQuestType<String>() {
+
+    override val elementFilter = "nodes with amenity = parcel_locker and !brand"
+    override val changesetComment = "Specify Parcel Locker brand"
```

Comment:
```suggestion
    override val changesetComment = "Specify parcel locker brand"
```

nouns are not capitalised by default in English